### ISSUE TITLE
Curated list of all camera's that work properly with Kerberos.io

### ISSUE THREAD

**Issue by cedricve** [kerberos-io/machinery#136]
Hey all,

I've started this thread because people always thinker if there camera will work properly with Kerberos.io, before the actually start buying hardware. My goal is to inform those people, and create some kind of "certification", so we can guarantee specific cameras work.

Please report your camera if it is missing from this list.

[x] Raspberry Pi Camera v1.3
[x] Raspberry Pi Camera v2

**Comment by palitu** [comment]
yeah, this is something that should be in their wiki page / docs.  It is one of my unknowns, what hardware to get?

**Comment by makama80** [comment]
Hello Cédric, I am using an el-cheapo Wanscam HW0026 WiFi IP camera for quite a few weeks now. To my knowledge it works fine, so it was time to report it here.. I also have a RPi now, but it works also fine with Armbian.

Kind regards,

Martijn

**Comment by wioxjk** [comment]
Curious if Axis M3065-V works too

**Comment by MKE1138** [comment]
I've got a C270 but Kerberos doesn't recognize it. Is there something that needs to be done to enable it?
